src/Compilers/Core/Portable/StrongName/StrongNameProvider.cs

```diff
@@ -40,6 +37,6 @@ protected StrongNameProvider()
         /// <summary>
         /// Create a <see cref="StrongNameKeys"/> for the provided information.
         /// </summary>
-        internal abstract StrongNameKeys CreateKeys(string keyFilePath, string keyContainerName, bool hasCounterSignature, CommonMessageProvider messageProvider);
+        internal abstract StrongNameKeys CreateKeys(string? keyFilePath, string? keyContainerName, bool hasCounterSignature, CommonMessageProvider messageProvider);
```

Comment:
Seeing this API get annotated brings back memories of trying to figure all this out (way back when). This would've made it so much easier.